eox_tenant/tenant_wise/__init__.py

```diff
@@ -3,14 +3,17 @@
 ==================
 This module makes it possible to override the some platform Models using new proxy models.
 """
+from eox_tenant.edxapp_wrapper.certificates_module import get_certificates_models
 from eox_tenant.edxapp_wrapper.site_configuration_module import get_site_configuration_models
-from eox_tenant.tenant_wise.proxies import TenantSiteConfigProxy
+from eox_tenant.tenant_wise.proxies import TenantSiteConfigProxy, TenantGeneratedCertificateProxy
 
 SiteConfigurationModels = get_site_configuration_models()
+CertificatesModels = get_certificates_models()
```

Comment:
If we are only loading the model here to use it below, why not combine both lines into one?